### PR TITLE
docs: GCS Create Bucket plugin doc

### LINKED ISSUE #15: DOC: Create a documentation for GCS Create Bucket plugin
Documentation for the GCS Create Bucket plugin is not present in docs, it needs to be created and added in docs.

### PR DESCRIPTION
**Pull request by bhushan-nemade-dt**
Fixes #15

### CODE REVIEW

**Review by abhibhaw — CHANGES_REQUESTED** [review]
No entry for this in index?

File: docs/user-guide/plugins/gcs-create-bucket.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Introduction
+The **GCS Create Bucket** plugin of Devtron enables automated creation of Google Cloud Storage (GCS) bucket directly within CI/CD workflows. By integrating the **GCS Create Bucket** plugin teams can simplify cloud storage provisioning and can efficiently manage and store application logs, deployment artifacts, backup data, and other critical application assets in a centralized cloud storage.
+
+### Prerequisites
+Before integrating the **GCS Create Bucket** plugin, ensure you have a Google Cloud Platform (GCP) account and GCP Project with appropriate permissions.
+
+---
+
+## Steps
+1. Go to **Applications** → **Devtron Apps**.
```
Comment (abhibhaw) [inline docs/user-guide/plugins/gcs-create-bucket.md]: Always think if the plugin will suit in job or app?

**Review by abhibhaw — CHANGES_REQUESTED** [review]

**Review by abhibhaw — DISMISSED** [review]
LGTM

File: docs/user-guide/plugins/gcs-create-bucket.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+| UniformAccess            | STRING       | Enable uniform bucket-level access control to create bucket with bucket level permissions instead of Access control list (true/false)            |  true            | 
+        
+### Trigger/Skip Condition
+Here you can set conditions to execute or skip the task. You can select `Set trigger conditions` for the execution of a task or `Set skip conditions` to skip the task.
+
+### Output Variables
+
+| Variable                 | Format       | Description |
+| ------------------------ | ------------ | ----------- | 
+|   BucketName             | STRING       | The name of the bucket createed.
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/gcs-create-bucket.md]: ~createed~ It should be 'created'

File: docs/user-guide/plugins/gcs-create-bucket.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# GCS Create Bucket
+
+## Introduction
+The **GCS Create Bucket** plugin of Devtron enables automated creation of Google Cloud Storage (GCS) bucket directly within CI/CD workflows. By integrating the **GCS Create Bucket** plugin teams can simplify cloud storage provisioning and can efficiently manage and store application logs, deployment artifacts, backup data, and other critical application assets in a centralized cloud storage.
```
Comment (ashoknayak777) [inline docs/user-guide/plugins/gcs-create-bucket.md]: Comma expected after `By integrating the **GCS Create Bucket** plugin`

**Review by ashoknayak777 — CHANGES_REQUESTED** [review]
@bhushan-nemade-dt Added a few suggestions

**Review by ashoknayak777 — APPROVED** [review]
LGTM

**Review by abhibhaw — APPROVED** [review]
LGTM

**Comment by abhibhaw** [comment]
@ashokdevtron I think we should put restriction on merging for this repo. To only allow squash cc: @prakarsh-dt

**Pull request merged**